plugins/dependencies/src/main/kotlin/de/fayard/refreshVersions/BuildSrcLibsTask.kt

```diff
@@ -100,39 +100,38 @@ open class BuildSrcLibsTask : DefaultTask() {
 }
 
 internal fun Project.findHardcodedDependencies(): List<Pair<Project, Configuration>> {
-    val versionsProperties = RefreshVersionsConfigHolder.readVersionProperties()
+    val versionsProperties = RefreshVersionsConfigHolder.readVersionsMap()
     val projectsWithHardcodedDependenciesVersions: List<Project> = rootProject.allprojects.filter {
         it.countDependenciesWithHardcodedVersions(versionsProperties) > 0
     }
 
     return projectsWithHardcodedDependenciesVersions.flatMap { project ->
         project.configurations.filterNot { configuration ->
-            configuration.shouldBeIgnored() || 0 == configuration.countDependenciesWithHardcodedVersions(versionsProperties, versionKeyReader)
+            configuration.shouldBeIgnored() || 0 == configuration.countDependenciesWithHardcodedVersions(versionsProperties, RefreshVersionsConfigHolder.versionKeyReader)
```

Comment:
Would be better to put these arguments on separate lines for readability I think.